pkg/bundle/bundle.go

```diff
@@ -136,22 +143,33 @@ func (b *ProtobufBundle) VerificationContent() (verify.VerificationContent, erro
 
 	switch content := b.VerificationMaterial.GetContent().(type) {
 	case *protobundle.VerificationMaterial_X509CertificateChain:
-		certs := content.X509CertificateChain.GetCertificates()
-		certificates := make([]*x509.Certificate, len(certs))
-		var err error
-		for i, cert := range content.X509CertificateChain.GetCertificates() {
-			certificates[i], err = x509.ParseCertificate(cert.RawBytes)
+		var parsedCert *x509.Certificate
+		for i, eachCert := range content.X509CertificateChain.GetCertificates() {
+			thisCert, err := x509.ParseCertificate(eachCert.RawBytes)
 			if err != nil {
 				return nil, ErrValidationError(err)
 			}
+
+			if i == 0 {
+				parsedCert = thisCert
```

Comment:
nit, can we break here, or just grab the first certificate in the chain if `len(content.X509CertificateChain.GetCertificates() > 0`? Otherwise there's a DoS vector where we parse an unlimited number of certificates when we only ingest the first.